apprating/src/main/java/com/mindera/skeletoid/apprating/utils/DateUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.mindera.skeletoid.apprating.utils
+
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+import java.util.TimeZone
+import java.util.concurrent.TimeUnit
+
+object DateUtils {
```

Comment:
DateUtils should be added to the base module and this one can depend on that one.